src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+body {
+  display: flex;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  text-transform: uppercase;
```

Comment:
Do not change this styles for whole page! Chage only where you need it
```suggestion
```

Reply:
done